packages/common/dto/form.dto.ts

```diff
@@ -0,0 +1,17 @@
+import { IsIn, IsString, ValidateNested } from 'class-validator';
+import { AvailabilityDTO } from './availability-information.dto';
+import { PersonalInformationDTO } from './personal-information.dto';
+import { SkillInformationDTO } from './skill-information.dto';
+
+export class FormDTO {
+  @ValidateNested()
+  payload!: {
```

Comment:
If possible it'd be nice to sync this naming up with the FE, not sure if we want to add a suffix/prefix but I'm open to changing the FE as long as we can be consistent. Particularly having personal/contact information separate would help

- \<stepName\>Information
- \<stepName\>Page
- etc..

Reply:
I believe the FE needs to align with the RESTfulness of an API. So, having the terminology contextual is the way6 to go compared to what is being used in the FE.

I don't want us to create a tightly coupled FE + BE just so we can have common entities and classes.